src/routes/ar-io.ts

```diff
@@ -203,13 +203,18 @@ arIoRouter.post(
   express.json(),
   async (req, res) => {
     try {
-      const { id } = req.body;
+      const { id, bypassFilter = true } = req.body;
```

Comment:
implemented to original request, but we could also make `isPrioritized` available as a body param as well to validate that behavior

Reply:
I think we can leave off `isPrioritized` for now. I am pondering another change though related to how we're queuing these.